panoramapublic/src/org/labkey/panoramapublic/PanoramaPublicFileImporter.java

```diff
@@ -61,12 +61,6 @@ public void process(@Nullable PipelineJob job, FolderImportContext ctx, VirtualF
             return;
         }
 
-        if (null == job)
-        {
-            log.error("Pipeline job not found.");
-            return;
-        }
-
```

Comment:
I see. This check was for a specific panorama work flow to copy a folder. Looks like for some folder imports, like container from template, the job is null. In that case the folder importer will roll through all these importers with a null job, so that would be an expected state. So removing this makes sense.